### DISCUSSION TITLE
How to find statistics

### DISCUSSION THREAD

**Discussion by Christian79AD** [rcaelers/workrave#392] [Q&A] [answered]
How can I see the statistics? Are they available in the Windows Version?

I use Workrave (workrave-win32-v1.10.49) on Windows 10.

**Comment by kees-jan** [comment, marked as answer]
Have you tried right clicking on the workrave window (or the applet, or the tray icon), and then selecting "statistics" from the menu that appears?

**Reply by Christian79AD** [reply]
Thanks! Right-clicking on the workrave window or tray icon shows the menu point "Statistics".

Before I looked only in the preferences.
